aspnetcore/security/authentication/accconfirm.md

```diff
@@ -86,128 +74,113 @@ You generally want to prevent new users from posting any data to your web site b
 
-> [!NOTE]
-> We're also preventing a newly registered user from being automatically logged on by commenting out the following line:
+Note: We're also preventing a newly registered user from being automatically logged on by commenting out the following line:
 >
 > ```csharp
 > //await _signInManager.SignInAsync(user, isPersistent: false);
 > ```
 
 *  Enable password recovery by uncommenting the code in the `ForgotPassword` action in the *Controllers/AccountController.cs* file.
 
-[!code-csharp[Main](accconfirm/sample/WebApplication3/src/WebApplication3/Controllers/AccountController.cs?highlight=17-21&range=272-297)]
+[!code-csharp[Main](accconfirm/sample/WebApp1/Controllers/AccountController.cs?highlight=17-23&name=snippet_ForgotPassword)]
 
-Uncomment the highlighted `ForgotPassword` from in the *Views/Account/ForgotPassword.cshtml* view file.
+Uncomment the markup in *Views/Account/ForgotPassword.cshtml*:
 
-[!code-html[Main](accconfirm/sample/WebApplication3/src/WebApplication3/Views/Account/ForgotPassword.cshtml?highlight=11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27)]
+[!code-html[Main](accconfirm/sample/WebApp1/Views/Account/ForgotPassword.cshtml)]
 
 ## Register, confirm email, and reset password
 
 In this section, run the web app and show the account confirmation and password recovery flow.
 
-* Run the application and register a new user
+* Run the app and register a new user
 
-![Web application Account Register view](accconfirm/_static/loginaccconfirm1.png)
+ ![Web application Account Register view](accconfirm/_static/loginaccconfirm1.png)
 
 * Check your email for the account confirmation link. If you don't get the email notification:
 
   * Check the SendGrid web site to verify your sent mail messages.
-
   * Check your spam folder.
-
   * Try another email alias on a different email provider (Microsoft, Yahoo, Gmail, etc.)
-
   * In SSOX, navigate to **dbo.AspNetUsers** and delete the email entry and try again.
 
 * Click the link to confirm your email.
-
 * Log in with your email and password.
-
 * Log off.
 
 ### Test password reset
 
-* Login and select **Forgot your password?**
-
+* If you're logged in, select **Logout**.  
+* Select the **Log in** link and select the **Forgot your password?** link.
 * Enter the email you used to register the account.
-
 * An email with a link to reset your password will be sent. Check your email and click the link to reset your password.  After your password has been successfully reset, you can login with your email and new password.
 
 ## Require email confirmation before login
 
 With the current templates, once a user completes the registration form, they are logged in (authenticated). You generally want to confirm their email before logging them in. In the section below, we will modify the code to require new users have a confirmed email before they are logged in. Update the `[HttpPost] Login` action in the *AccountController.cs* file with the following highlighted changes.
```

Comment:
You're right. What this is really doing is not logging in a user once they register.